cogs/cipher.py

```diff
@@ -0,0 +1,111 @@
+import base64
+import hashlib
+import string
+
+from discord.ext import commands
+
+from assets.context import Command, argument, example, usage_info
+
+
+class Ciphers(commands.Cog):
+    character_set = [string.ascii_lowercase, string.ascii_uppercase, string.digits]
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def cog_check(self, ctx):
```

Comment:
WO0236 : Method "cog_check" was expected to be non-async, found it instead as async (invalid-overriden-method)